hawkular-alerters/hawkular-prometheus-alerter/hawkular-prometheus-alerter-itest/src/test/groovy/org/hawkular/alerter/prometheus/PrometheusITest.groovy

```diff
@@ -84,11 +82,10 @@ class prometheusITest {
         assertEquals(200, response.status)
     }
 
-    @Ignore
     @Test
     void prometheusAlerterAlertTest() {
         def start = String.valueOf(System.currentTimeMillis())
-        // load external trigger
+        // load external group trigger
```

Comment:
Minor, I think this test is using prometheus-alert-trigger.json which doesn't use group trigger, so the comment led me to think the json file was wrong, but I think is the comment which is bad.